frontend/src/components/ScheduleDialog/ScheduleDialog.js

```diff
@@ -25,175 +22,122 @@ import {
-                                )}
-                            </div>
-                            <Autocomplete
-                                {...defaultProps}
-                                id="group"
-                                clearOnEscape
-                                openOnFocus
-                                className="room-field"
-                                onChange={(event, newValue) => {
-                                    setRoom(newValue);
-                                }}
-                                renderInput={(params) => (
-                                    <TextField
-                                        {...params}
-                                        label={translation(FORM_ROOM_LABEL)}
-                                        margin="normal"
-                                    />
-                                )}
-                            />
-                            <div className="buttons-container">
-                                <Button
-                                    className="dialog-button"
-                                    variant="contained"
-                                    color="primary"
-                                    onClick={() => chooseClickHandle()}
-                                >
-                                    {translation(FORM_CHOOSE_BUTTON_TITLE)}
-                                </Button>
-                                <Button
-                                    className="dialog-button"
-                                    variant="contained"
-                                    onClick={() => onClose()}
-                                >
-                                    {translation(FORM_CANCEL_BUTTON_TITLE)}
-                                </Button>
-                            </div>
-                        </>
-                    )}
-                </>
-            ) : (
-                <>
-                    <DialogTitle id="simple-dialog-title">
-                        <p className="availability-warning">
-                            {!room.available ? `${translation(COMMON_ROOM_IS_UNAVAILABLE)}. ` : ''}
-                        </p>
-                        <p className="availability-warning">
-                            {!availability.teacherAvailable
-                                ? `${translation(COMMON_TEACHER_IS_UNAVAILABLE)}. `
-                                : ''}
-                        </p>
-
-                        <p className="availability-warning">
-                            {!availability.classSuitsToTeacher
-                                ? `${translation(COMMON_CLASS_DOES_NOT_SUIT_FOR_TEACHER)}. `
-                                : ''}
-                        </p>
-
-                        {translation(COMMON_ARE_YOU_SURE)}
-                    </DialogTitle>
-                    <div className="buttons-container">
+        <>
+            <CustomDialog
```

Comment:
Shouldn`t it be like in line 107?

Reply:
Not as we perform the check for ScheduleDialog externally